app/answerOperations.js

```diff
@@ -0,0 +1,58 @@
+var Question = require('../models/question');
+var Answer = require('../models/answer');
+var textUtils = require('../utils/textutils');
+var tokenUtils = require('../utils/tokenutils');
+var Constants = require('../constants');
+
+module.exports = function () {
+    return setupFunctions();
+};
+
+var setupFunctions = function () {
+    var exports = {};
+
+    exports.postAnswer = function (req, res) {
+        var questionId = req.body.questionId;
+        if (textUtils.isEmpty(questionId)) {
```

Comment:
For these utility functionalities, lets use [lodash] ( https://lodash.com/docs ). 

Benefits:
- Dont need test coverage.
- Many, many more functions at our disposal. 
- One less point of failure.